devicetypes/smartthings/zwave-temp-humidity-sensor.src/zwave-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,213 @@
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Generic Z-Wave Water/Temp/Humidity Sensor
+ *
+ *  Author: SmartThings
+ *  Date: 2020-07-22
+ */
+
+metadata {
+	definition(name: "Z-Wave Temp/Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", "vid": "generic-humidity", ocfDeviceType: "oic.d.thermostat") {
+		capability "Temperature Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Battery"
+		capability "Sensor"
+		capability "Health Check"
+		capability "Configuration"
+
+		fingerprint mfr:"0371", prod:"0002", model:"0009", deviceJoinName: "aeorQ Multipurpose Sensor", mnmn: "SmartThings", "vid": "SmartThings-smartthings-Aeotec_Aerq_Temperature_Humidity_Sensor"	//EU // Aeotec Aerq Temperature and Humidity Sensor
+		fingerprint mfr:"0371", prod:"0102", model:"0009", deviceJoinName: "aeorQ Multipurpose Sensor", mnmn: "SmartThings", "vid": "SmartThings-smartthings-Aeotec_Aerq_Temperature_Humidity_Sensor"	//US // Aeotec Aerq Temperature and Humidity Sensor
+		fingerprint mfr:"0371", prod:"0202", model:"0009", deviceJoinName: "aeorQ Multipurpose Sensor", mnmn: "SmartThings", "vid": "SmartThings-smartthings-Aeotec_Aerq_Temperature_Humidity_Sensor"	//AU // Aeotec Aerq Temperature and Humidity Sensor
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "temperature", type: "generic", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.temperature", key: "PRIMARY_CONTROL") {
+				attributeState "temperature", label: '${currentValue}°',
+					backgroundColors: [
+						[value: 31, color: "#153591"],
+						[value: 44, color: "#1e9cbb"],
+						[value: 59, color: "#90d2a7"],
+						[value: 74, color: "#44b621"],
+						[value: 84, color: "#f1d801"],
+						[value: 95, color: "#d04e00"],
+						[value: 96, color: "#bc2323"]
+					]
+			}
+		}
+		valueTile("humidity", "device.humidity", inactiveLabel: false, width: 2, height: 2) {
+			state "humidity", label: '${currentValue}% humidity', unit: ""
+		}
+		valueTile("dewPoint", "device.dewPoint", inactiveLabel: false, width: 2, height: 2) {
+			state "dewPoint", label: '${currentValue}° dewPoint', unit: ""
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", label: "", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "temperature", "humidity", "dewPoint"
+		details(["temperature", "humidity", "dewPoint", "battery"])
+	}
+}
+
+def updated() {
+    configure()
+}
+
+def installed() {
+    sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 10 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
Was checkInterval set in accordance to default wakeUpInterval of those devices ?

Reply:
@KKlimczukS according to our discussion with @PKacprowiczS i will leave this checkInterval time as a default period for sleepy devices